openedx/adg/common/mailchimp_pipeline/tests/test_helpers.py

```diff
@@ -36,5 +36,5 @@ def test_get_enrollment_course_names_and_short_ids_by_user(user_enrollments):  #
     user, _ = user_enrollments
     course_short_ids, course_titles = get_enrollment_course_names_and_short_ids_by_user(user)
 
-    assert course_short_ids == '100, 101, 102'
-    assert course_titles == 'course1, course2, course3'
+    assert course_short_ids == '100,101,102'
+    assert course_titles == 'course1,course2,course3'
```

Comment:
It is fixed in @annas-usman's [PR](https://philanthropyu.atlassian.net/browse/LP-2458) as well, so we would have to revert from one file, and use we can manually check that this particular test is failing due to this issue and would be fixed

Reply:
Okay then I am reverting this one.